### ISSUE TITLE
TX Attr TODO

### ISSUE THREAD

**Issue by jdinan** [Sandia-OpenSHMEM/SOS#148]
In transport_ofi.c, is the following TODO still needed?

```
    hints.tx_attr = &tx_attr; /* TODO: fill tx_attr */
```

If (true) create an issue for it.  Else, please remove the TODO.

**Comment by jdinan** [comment]
@kseager -- Already done, keep for 1.3, or push to 1.3.1?

**Comment by kseager** [comment]
I think we should push to 1.3.1, this more-so concerns filling out the ordering aspect...

**Comment by jdinan** [comment]
@kseager Is this done?

**Comment by kseager** [comment]
this is waiting on OFI ordering semantics finalization

**Comment by jdinan** [comment]
@kseager Any update?
